base_station/gui/src/components/PDBFuse.vue

```diff
@@ -0,0 +1,305 @@
+        </tr>
+        <tr>
+          <td class="tableElement">Cytron 6</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.cytron6.current.color}">{{fuse_data.cytron6.current.val.toFixed(5)}}A</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.cytron6.voltage.color}">{{fuse_data.cytron6.voltage.val.toFixed(5)}}V</td>
+        </tr>
+        <tr>
+          <td class="tableElement">Nichrome 1</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.nichrome1.current.color}">{{fuse_data.nichrome1.current.val.toFixed(5)}}A</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.nichrome1.voltage.color}">{{fuse_data.nichrome1.voltage.val.toFixed(5)}}V</td>
+        </tr>
+        <tr>
+          <td class="tableElement">Nichrome 2</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.nichrome2.current.color}">{{fuse_data.nichrome2.current.val.toFixed(5)}}A</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.nichrome2.voltage.color}">{{fuse_data.nichrome2.voltage.val.toFixed(5)}}V</td>
+        </tr>
+        <tr>
+          <td class="tableElement">Nichrome 3</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.nichrome3.current.color}">{{fuse_data.nichrome3.current.val.toFixed(5)}}A</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.nichrome3.voltage.color}">{{fuse_data.nichrome3.voltage.val.toFixed(5)}}V</td>
+        </tr>
+        <tr>
+          <td class="tableElement">HBridge 1</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.hbridge1.current.color}">{{fuse_data.hbridge1.current.val.toFixed(5)}}A</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.hbridge1.voltage.color}">{{fuse_data.hbridge1.voltage.val.toFixed(5)}}V</td>
+        </tr>
+        <tr>
+          <td class="tableElement">HBridge 2</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.hbridge2.current.color}">{{fuse_data.hbridge2.current.val.toFixed(5)}}A</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.hbridge2.voltage.color}">{{fuse_data.hbridge2.voltage.val.toFixed(5)}}V</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      pdb_data: {
+        three: {
+          temp: {val: 0.0, color: "grey"},
+          current: {val: 0.0, color: "grey"},
+          voltage: {val: 0.0, color: "grey"}
+        },
+        five: {
+          temp: {val: 0.0, color: "grey"},
+          current: {val: 0.0, color: "grey"},
+          voltage: {val: 0.0, color: "grey"}
+        },
+        twelve: {
+          temp: {val: 0.0, color: "grey"},
+          current: {val: 0.0, color: "grey"},
+          voltage: {val: 0.0, color: "grey"}
+        }
+      },
+
+      fuse_data: {
+        cytron1: {
```

Comment:
can these be changed to make more sense to an operator not as familar w the hardware? like joint A, joint B, etc, even in the SA config we refer to the joints by the same names (i.e joint A, B, D, etc)